includes/forms/class-llms-forms.php

```diff
@@ -472,7 +473,22 @@ public function get_fields_settings_from_blocks( $blocks ) {
 
 		foreach ( $blocks as $block ) {
 			$settings = $this->block_to_field_settings( $block, $blocks );
-			if ( $settings ) {
+
+			if ( empty( $settings ) ) {
+				continue;
+			}
+			if (
+				'hidden' === ( $settings['type'] ?? null ) &&
+				in_array( $block['attrs']['field'] ?? null, array( 'checkbox', 'radio' ) )
```

Comment:
For some reason a suggestion I made was not included in the review... why? dunno.

basically, we don't need any special treatment for radio buttons as far as I can tell (am I wrong?), as the `radio` value isn't an array.

```suggestion
				isset( $block['attrs']['field'] ) && 'checkbox' === $blocks['attrs']['field'] )
```

If you think I'm wrong and `radio` should be included then you should do a strict type comparison here (cs):
```
				in_array( $block['attrs']['field'] ?? null, array( 'checkbox', 'radio' ), true )
```

Reply:
@eri-trabiccolo, you are correct! I saw that radio blocks could have an array of options, but didn't think about the "name" attribute not using the `[]` array trick. Thanks for catching this!